demo-kotlin/app/src/main/java/com/example/placesdemo/CurrentPlaceActivity.kt

```diff
@@ -41,6 +42,30 @@ class CurrentPlaceActivity : AppCompatActivity() {
     private lateinit var responseView: TextView
     private lateinit var fieldSelector: FieldSelector
 
+    @SuppressLint("MissingPermission")
+    val requestPermissionLauncher =
+        registerForActivityResult(
+            ActivityResultContracts.RequestMultiplePermissions()
+        ) { permissions ->
+            when {
+                permissions[permission.ACCESS_FINE_LOCATION] == true -> {
+                    // Precise location access granted.
+                    findCurrentPlaceWithPermissions()
+                }
+                permissions[permission.ACCESS_COARSE_LOCATION] == true -> {
+                    // Only approximate location access granted.
+                    findCurrentPlaceWithPermissions()
+                }
+                else -> {
+                    Toast.makeText(
+                        this,
+                        "Both ACCESS_COARSE_LOCATION & ACCESS_FINE_LOCATION permissions are required",
```

Comment:
I believe this should be "Either ACCESS_FINE_LOCATION or ACCESS_COARSE_LOCATION permission is required.", per [L142](https://github.com/googlemaps/android-places-demos/pull/348/files#diff-ffc8f0109e203da82714446ad27d274277c3e2c5893e110e0f6a6fe72bd9ecf2R142)

Reply:
Jep, I have changed it